org.flameshot.Flameshot.yml

```diff
@@ -1,26 +1,23 @@
 app-id: org.flameshot.Flameshot
 runtime: org.kde.Platform
-runtime-version: 5.15-21.08
+runtime-version: 5.15-23.08
 sdk: org.kde.Sdk
 command: flameshot
 finish-args:
-  # X11 + XShm access
+  # Fallback X11 + XShm access
   - --share=ipc
-  - --socket=x11
+  - --socket=fallback-x11
   # Wayland access
   - --socket=wayland
   - --device=dri
   # Connectivity
   - --share=network
   # QtSingleApplication, allow other instances to see log files
   - --env=TMPDIR=/var/tmp
-  # Allow loading/saving files from anywhere
-  - --filesystem=host
   # Notification access
   - --talk-name=org.freedesktop.Notifications
   # System Tray Icon
   - --talk-name=org.kde.StatusNotifierWatcher
-  - --own-name=org.kde.*
```

Comment:
There seems to be some issues with the tray icon on GNOME. The icon shows up for a while but then immediately changes the cross/blank icon.

Giving back access to `--own-name=org.kde.*` or `--filesystem=host` does not seem to fix this, so there's something else going wrong.

This should probably be investigated.

Reply:
[4293419e16.webm](https://github.com/flathub/org.flameshot.Flameshot/assets/62639087/b0e0ca73-a9c6-41e2-b5b4-90a292ce4daf)

Reply:
Ah hrm this is just https://github.com/flathub/org.flameshot.Flameshot/issues/18